dbms/src/Storages/Page/V2/CMakeLists.txt

```diff
@@ -42,16 +42,4 @@ add_library(page_storage_v2 EXCLUDE_FROM_ALL
 )
 target_include_directories(page_storage_v2 PUBLIC ${TiFlash_SOURCE_DIR}/contrib/tiflash-proxy/raftstore-proxy/ffi/src)
 target_link_libraries(page_storage_v2 clickhouse_common_io cpptoml
-        kv_client tipb) # TODO: remove dependency on these libs. Now we need them for DB::Context
-
-### Build a control binary for PageStorage
-## For `page_ctl`, we need to define `PAGE_STORAGE_UTIL_DEBUGGGING`
-add_executable(page_ctl EXCLUDE_FROM_ALL
-        tests/page_storage_ctl.cpp
-        ${page_storage_v2_headers} ${page_storage_v2_sources}
-        ${io_base_headers} ${io_base_sources}
-)
```

Comment:
Need to update our CI script to remove `page_ctl` from unit-test build target
https://github.com/PingCAP-QE/ci/blob/main/jenkins/pipelines/ci/tiflash/tiflash-build-common.groovy#L136-L138